crates/ruff_python_formatter/src/expression/expr_ellipsis_literal.rs

```diff
@@ -0,0 +1,33 @@
+use ruff_python_ast::AnyNodeRef;
+use ruff_python_ast::ExprEllipsisLiteral;
+
+use crate::comments::SourceComment;
+use crate::expression::parentheses::{NeedsParentheses, OptionalParentheses};
+use crate::prelude::*;
+
+#[derive(Default)]
+pub struct FormatExprEllipsisLiteral;
+
+impl FormatNodeRule<ExprEllipsisLiteral> for FormatExprEllipsisLiteral {
+    fn fmt_fields(&self, _item: &ExprEllipsisLiteral, f: &mut PyFormatter) -> FormatResult<()> {
+        token("...").fmt(f)
+    }
+
+    fn fmt_dangling_comments(
+        &self,
+        _dangling_comments: &[SourceComment],
+        _f: &mut PyFormatter,
+    ) -> FormatResult<()> {
+        Ok(())
+    }
```

Comment:
We shouldn't need those, these constants can't have dangling comments, iirc that's only a thing for implicitly concatenated strings where comments can be placed like:
```python
a = (
"1"
# comment
"2"
)